internal/buildscripts/packaging/fpm/etc/otel/collector/splunk-support-bundle.ps1

```diff
@@ -0,0 +1,252 @@
+#  - OUTPUTS: None
+#  - RETURN: 0
+#######################################
+function getMetrics {
+    Write-Host "INFO: Getting metric information..."
+    $connection = New-Object System.Net.Sockets.TcpClient("localhost", 8888)
+    if ($connection.Connected) {
+        cmd.exe /c "curl -s http://localhost:8888/metrics > $TMPDIR/metrics/collector-metrics.txt 2>&1"
+    } else { 
+        Write-Host "WARN: localhost:8888/metrics unavailable so metrics not collected"
+    }
+}
+
+#######################################
+# Gather zpages
+#  - GLOBALS: TMPDIR
+#  - ARGUMENTS: None
+#  - OUTPUTS: None
+#  - RETURN: 0
+#######################################
+function getZpages {
+    Write-Host "INFO: Getting zpages information..."
+    $connection = New-Object System.Net.Sockets.TcpClient("localhost", 55679)
+    if ($connection.Connected) {
+        cmd.exe /c "curl -s http://localhost:55679/debug/tracez > $TMPDIR/zpages/tracez.html 2>&1"
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+        $packages = Invoke-WebRequest -Uri "http://localhost:55679/debug/tracez" -UseBasicParsing
+        foreach ($package in $packages.links.href) {
+            $ENCODED_PACKAGE_NAME = [System.Web.HTTPUtility]::UrlEncode("$package")
+            cmd.exe /c "curl -s `"http://localhost:55679/debug/$package`" > $TMPDIR/zpages/$global:DIRECTORY/debug/$ENCODED_PACKAGE_NAME 2>&1"
+        }
+    } else { 
+
+        Write-Host "WARN: localhost:55679 unavailable so zpages not collected"
+    }
+
+        
+}
+
+#######################################
+# Gather System information
+#  - GLOBALS: TMPDIR
+#  - ARGUMENTS: None
+#  - OUTPUTS: None
+#  - RETURN: 0
+#######################################
+function getHostInfo {
+    Write-Host "INFO: Getting host information..."
+    for ( $i = 0; $i -lt 3; $i++ ) {
+        Get-Process -Name 'otelcol' >> $TMPDIR/metrics/top.txt 2>&1 
+        Get-Process -Name 'fluentd' >> $TMPDIR/metrics/top.txt 2>&1 
+        Start-Sleep -s 2
+    }
+    if (-NOT (Get-Process -Name 'otelcol')) {
+        Write-Host "WARN: Unable to find otelcol PIDs"
+        Write-Host "      top will not be collected for otelcol";
+    }
+    if (-NOT (Get-Process -Name 'fluentd')) {
+        Write-Host "WARN: Unable to find fluentd PIDs"
+        Write-Host "      top will not be collected for fluentd";
+    }
+    
+    Get-PSDrive > $TMPDIR/metrics/df.txt 2>&1
+    
+    Get-CIMInstance Win32_OperatingSystem | Select TotalVisibleMemorySize,FreePhysicalMemory,TotalVirtualMemorySize,FreeVirtualMemory > $TMPDIR/metrics/free.txt 2>&1
+}
+
+#######################################
+# Tar support bundle
+#  - GLOBALS: TMPDIR
+#  - ARGUMENTS: None
+#  - OUTPUTS: None
+#  - RETURN: 0 if successful, non-zero on error
+#######################################
+function tarResults {
+    Write-Host "INFO: Creating tarball..."
+    $TAR_NAME = Split-Path $TMPDIR -leaf
+    tar -cf "$TAR_NAME.tar.gz" $TMPDIR 2>&1

```

Comment:
This is only supported in recent versions of Windows -- we will need compatibility for every version we support

Reply:
We have changed the tar command with below mentioned command:
```
    Add-Type -assembly "system.io.compression.filesystem"
    [io.compression.zipfile]::CreateFromDirectory($TMPDIR, "$TAR_NAME.tar.gz")
```
We have checked with Windows Server 2012, Windows Server 2016 and, Windows Server 2019; It is working fine.
Attaching the screenshots for reference.
![image](https://user-images.githubusercontent.com/89311282/130463744-fcc6f3db-8333-41d8-8347-3f1c9943149f.png)

![image](https://user-images.githubusercontent.com/89311282/130463803-bfd41817-39b7-4aab-a279-66b07969ad90.png)

![image](https://user-images.githubusercontent.com/89311282/130463833-3b8b899b-dc6b-4b74-9e8a-4880ecf3714f.png)

Kindly let us know if any further changes required.